docker/redis.conf

```diff
@@ -0,0 +1,47 @@
+################################## NETWORK #####################################
+bind 127.0.0.1
+protected-mode yes
+port 6379
+tcp-backlog 511
+timeout 0
+tcp-keepalive 300
+
+################################# GENERAL #####################################
+daemonize yes
+supervised no
+pidfile redis.pid
+loglevel notice
+logfile redis.log
+
+################################ SNAPSHOTTING  ################################
+save 900 1
```

Comment:
Based on our discussion, disable this snapshotting for now. If the requirements exists to snapshot requests, we can re-do the changes.